pages/accountLists/[accountListId]/tools/GetAppeals.graphql

```diff
@@ -0,0 +1,15 @@
+# Write your query or mutation here
+query GetAppeals($accountListId: ID!) {
+  appeals(accountListId: $accountListId, primary: false) {
+    nodes {
+      id
+      name
+      amount
+      amountCurrency
+      pledgesAmountNotReceivedNotProcessed
+      pledgesAmountReceivedNotProcessed
+      pledgesAmountProcessed
+      pledgesAmountTotal
+    }
+  }
+}
```

Comment:
I think you can fetch everything you need in a single query instead of firing off 2 at the same time:
```suggestion
query GetAppeals($accountListId: ID!) {
  nonPrimaryAppeals: appeals(accountListId: $accountListId, primary: false) {
    nodes {
      ...AppealFields
    }
  }
  primaryAppeals: appeals(
    accountListId: $accountListId
    primary: true
    first: 1
  ) {
    nodes {
      ...AppealFields
    }
  }
}

fragment AppealFields on Appeal {
  id
  name
  amount
  amountCurrency
  pledgesAmountNotReceivedNotProcessed
  pledgesAmountReceivedNotProcessed
  pledgesAmountProcessed
  pledgesAmountTotal
}
```
Idk what to call `nonPrimaryAppeals`. `otherAppeals`? Just `appeals`? You can pick a name.